sklearn/preprocessing/tests/test_label.py

```diff
@@ -35,16 +36,26 @@ def toarray(a):
 
 
 def test_label_binarizer():
-    lb = LabelBinarizer()
-
     # one-class case defaults to negative label
-    inp = ["pos", "pos", "pos", "pos"]
+    # For dense case:
+    lb = LabelBinarizer(sparse_output=False)
     expected = np.array([[0, 0, 0, 0]]).T
+    inp = ["pos", "pos", "pos", "pos"]
```

Comment:
(nitpick) I think we can move this line above 41 and reuse for both the tests...